package/psyneulinkviewer/node.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import subprocess
+import logging
+import configuration
+from setuptools.command.install import install
+from packaging.version import Version
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+def install_node():
+    logging.info("Installing node")
+    subprocess.run(configuration.conda_forge, shell=True)
+    subprocess.run(configuration.node_installation, shell=True)
+
+def check_node_installation():
+    try:
+        node_version = subprocess.run(
```

Comment:
the same raised here https://github.com/MetaCell/PsyNeuLinkView/pull/155#discussion_r1705255716 applies to this code as well. Thanks

Reply:
Fix applied